app/src/main/res/layout/view_fireproof_website_empty_hint.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) 2020 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingTop="24dp"
+    android:paddingStart="16dp"
+    android:paddingEnd="16dp"
+    android:paddingBottom="24dp">
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/fireproofWebsiteEmptyHint"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:fontFamily="sans-serif"
+        android:lineSpacingExtra="4sp"
+        android:gravity="center"
+        android:textColor="?attr/bookmarkTitleTextColor"
+        android:textSize="16sp"
+        android:textStyle="normal"
+        tools:text="Lorem ipsum dolor sit amet" />
```

Comment:
missing empty line after closure

Reply:
I formatted the file from AS but no new lines are added. Is that something I don't have correctly configured? is there any guideline I may be missing?

Reply:
did you import the `settings.jar` as per [guidelines](https://github.com/duckduckgo/Android/blob/develop/styleguide/STYLEGUIDE.md)?